cmds/rm/rm.go

```diff
@@ -19,16 +19,15 @@ import (
 	"flag"
 	"fmt"
 	"os"
-	"path"
+	"path/filepath"
 )
 
 var (
-	flags struct {
-		r bool
-		v bool
-		i bool
-	}
```

Comment:
To still support this use case, we should kind of go back to the original way this was defined.

You can either make flags a struct again, or you could do this:

```go
var recursive bool

func init() {
  flag.BoolVar(&recursive, "r", false, "remove files recursively")
  flag.BoolVar(&recursive, "R", false, "remove files recursively")
  flag.BoolVar(&recursive, "recursive", false, "remove files recursively")
}
```